### PR TITLE
media-gfx/frogr: update ebuild

### PR DESCRIPTION
**Pull request by cfuga**
Added dev-util/itstool as BDEPEND, sort deps, bump to version 1.7.

Signed-off-by: Cristian Othón Martínez Vera <cfuga@cfuga.mx>
Bug: https://bugs.gentoo.org/649234

### CODE REVIEW

File: media-gfx/frogr/frogr-1.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+BDEPEND="
+	dev-util/itstool
+	>=sys-devel/gettext-0.19.7
+	virtual/pkgconfig
+"
+# TODO add a useflag for enable-video or header-bar???
+
+pkg_postinst() {
+	xdg_pkg_postinst
+	gnome2_icon_cache_update
```
Comment (juippis) [inline media-gfx/frogr/frogr-1.7.ebuild:35]: This doesn't exist in the *gnome2-utils* eclass anymore, it's in *xdg-utils* and can be called with *xdg.eclass* that you inherit.

File: media-gfx/frogr/frogr-1.7.ebuild [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+HOMEPAGE="https://live.gnome.org/Frogr"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=dev-libs/glib-2.44:2
+	>=dev-libs/json-glib-1.2
+	>=dev-libs/libgcrypt-1.5:*
```
Comment (juippis) [inline media-gfx/frogr/frogr-1.7.ebuild:18]:
```suggestion
	>=dev-libs/libgcrypt-1.5:=
```

**Review by juippis — COMMENTED** [review]
Add `x11-libs/gdk-pixbuf:2` to RDEPEND too.

**Comment by cfuga** [comment]
> Add `x11-libs/gdk-pixbuf:2` to RDEPEND too.

Thanks! I'll apply all the fixes to version 1.7. After merging this PR to the main branch, I'll drop the previous version.

**Review by juippis — APPROVED** [review]
Thanks!

**Comment by juippis** [comment]
0c83acf47fb5a28e736db9038b318d365afbe094
89ddfe63d3ff7e91d338b6f53490895c43012e91
93a9d08a5e93bc1462cda89380abd587b44e2277
f4cbd515806c19991078646648307285f7a225cf

**Pull request closed without merging**